lib/internal/inspector_network_tracking.js

```diff
@@ -10,6 +13,17 @@ let Network;
 let requestId = 0;
 const getNextRequestId = () => `node-network-event-${++requestId}`;
 
+// Convert a Headers object (Map<string, number | string | string[]>) to a plain object (Map<string, string>)
+const headerObjectToDictionary = (headers = {}) => {
+  const dict = {};
+  for (const { 0: key, 1: value } of ObjectEntries(headers)) {
+    if (typeof value === 'string') dict[key] = value;
+    if (ArrayIsArray(value)) dict[key] = value.join(', ');
```

Comment:
`cookie` is a special header that uses `;` as value separator.

Reply:
Addressed e02aaa76caecbe1cf1278c43601dd8716a3558e9

<img width="503" alt="image" src="https://github.com/user-attachments/assets/533458f1-46e9-47df-a92d-b7de2c328fa4">